setup.py

```diff
@@ -49,7 +49,7 @@
         'flask-cors == 3.0.2',
         'gunicorn >= 19.7.1',
         'numpy >= 1.11, < 1.15',
-        'psutil == 5.4.2',
+        'psutil >= 5.4.6, < 6.0.0',
```

Comment:
Are we sure that `psutil` applies semantic versioning?

Reply:
I'm not sure, but I believe so, i.e from the changelog it seems API changes are grouped by majors:

- https://github.com/giampaolo/psutil/blob/master/HISTORY.rst#100
- https://github.com/giampaolo/psutil/blob/master/HISTORY.rst#200


But there's no reference to semantic versioning in the doc.

Safer option: `==5.4.6` ?

Reply:
I'll leave it up to your judgment, reading the other dependencies, it seems like we don't have a clear policy about that 🙂.